papers/deep_residual_learning/Deep_Residual_Learning_CIFAR-10.py

```diff
@@ -113,7 +113,7 @@ def residual_block(l, increase_dim=False, projection=False):
                 block = NonlinearityLayer(ElemwiseSumLayer([stack_2, projection]),nonlinearity=rectify)
             else:
                 # identity shortcut, as option A in paper
-                identity = ExpressionLayer(l, lambda X: X[:, :, ::2, ::2], lambda s: (s[0], s[1], s[2]//2, s[3]//2))
+                identity = ExpressionLayer(l, lambda X: X[:, :, ::2, ::2], lambda s: (s[0], s[1], int((s[2] + 0.5) // 2), int((s[3] + 0.5) // 2)))
```

Comment:
Wait, it should have been `(s[2]+1)//2`. This rounds up and doesn't need an `int` conversion either.